### PR TITLE
DDF-3155 Fixing a few minor issues in the hardening guide

### PR DESCRIPTION
**Pull request by brjeter**
#### Who is reviewing it? 
@paouelle @figliold 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/docs
#### Choose 2 committers to review/merge the PR. 
@ricklarsen - Documentation
@stustison
#### What are the relevant tickets?
[DDF-3155](https://codice.atlassian.net/browse/DDF-3155)
#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc [hunk 1]
```diff
@@ -78,5 +78,5 @@ Be sure to limit the hosts that are allowed to connect to the STS:
 
 * Open the `<${branding}_HOME>/etc/system.properties` file.
 * Edit the line `ws-security.subject.cert.constraints = .*`.
-** Remove the `.*` and replace with a comma-delimited list of desired hosts (`<MY_HOST>`):
-*** `ws-security.subject.cert.constraints = <MY_HOST>,<OTHER_HOST>`
+** Remove the `.*` and replace with a comma-delimited list of regular expressions for the DN's of all desired hosts (`<MY_HOST_DN>`):
```
Comment (figliold) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc]: ✏️  You shouldn't have an apostrophe in DN's.

File: distribution/docs/src/main/resources/_contents/_securing/configuring-web-context-policy-manager-contents.adoc [hunk 2]
```diff
@@ -73,5 +73,5 @@ Be sure to limit the hosts that are allowed to connect to the STS:
 
 * Open the `<${branding}_HOME>/etc/system.properties` file.
 * Edit the line `ws-security.subject.cert.constraints = .*`.
-** Remove the `.*` and replace with a comma-delimited list of desired hosts (`<MY_HOST>`):
-*** `ws-security.subject.cert.constraints = <MY_HOST>,<OTHER_HOST>`
+** Remove the `.*` and replace with a comma-delimited list of regular expressions for the DN's of all desired hosts (`<MY_HOST_DN>`):
```
Comment (figliold) [inline distribution/docs/src/main/resources/_contents/_securing/configuring-web-context-policy-manager-contents.adoc]: ✏️  You shouldn't have an apostrophe in DN's.

File: distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc [hunk 3]
```diff
@@ -134,7 +134,7 @@ For example:
 [NOTE]
 ====
 By default, the Catalog Backup Post-Ingest Plugin is *NOT* enabled.
-To enable, the Enable Backup Plugin configuration item must be checked in the Backup Post-Ingest Plugin configuration.
+To enable, the Enable Backup Plugin configuration item must be checked in the Backup Post-Ingest Plugin configuration under the Catalog app.
```
Comment (vinamartin) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc]: ❓  Is there a variable substitution that can be used for "Catalog app"?

File: distribution/docs/src/main/jdocs/content/_configuring/hardening-user-access-contents.adoc [hunk 4]
```diff
@@ -22,7 +22,6 @@ If a system recovery account is configured in `users.properties`, ensure:
 * The use of this account should be for as short a time as possible.
 * The default username/password of "`admin/admin`" should not be used.
 * All organizational standards for password complexity should still apply.
-* The password should be encrypted.
```
Comment (stustison) [inline distribution/docs/src/main/jdocs/content/_configuring/hardening-user-access-contents.adoc]: Why'd we remove this?

File: distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc (hunk 3 above)
Comment (brjeter) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc]: Fixed.

File: distribution/docs/src/main/resources/_contents/_securing/configuring-web-context-policy-manager-contents.adoc (hunk 2 above)
Comment (brjeter) [inline distribution/docs/src/main/resources/_contents/_securing/configuring-web-context-policy-manager-contents.adoc]: Fixed.

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc (hunk 1 above)
Comment (brjeter) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc]: Fixed

File: distribution/docs/src/main/jdocs/content/_configuring/hardening-user-access-contents.adoc (hunk 4 above)
Comment (brjeter) [inline distribution/docs/src/main/jdocs/content/_configuring/hardening-user-access-contents.adoc]: Changed to include steps.

**Comment by brjeter** [comment]
Comments addressed

**Comment by ricklarsen** [comment]
build now

**Pull request merged**